sechub-pds-solutions/iac/helm/pds-iac/templates/networkpolicy.yaml

```
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: MIT
+
+{{- if .Values.networkPolicy.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+    name: {{ .Chart.Name }}-policy
+spec:
+    podSelector:
+        matchLabels:
+            name: {{ .Chart.Name }}
+
+  {{- if .Values.networkPolicy.ingress }}
+    ingress:
+{{ .Values.networkPolicy.ingress | toYaml | indent 4 }}
```

This looks a bit nicer:
```suggestion
    {{ .Values.networkPolicy.ingress | toYaml | indent 4 | trim }}
```